nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/http/FilenameStrategy.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.nifi.processors.standard.http;
```

Comment:
This file is missing the license header.

Reply:
Thanks for noticing. I've added the missing header.